app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -208,7 +221,7 @@ class StateFragmentPresenter @Inject constructor(
   fun handleKeyboardAction() {
     hideKeyboard()
     if (viewModel.getCanSubmitAnswer().get() == true) {
-      handleSubmitAnswer(viewModel.getPendingAnswer(recyclerViewAssembler))
+      handleSubmitAnswer(viewModel.getPendingAnswer(answerHandlerRetriever))
```

Comment:
Can we use ``recyclerViewAssembler::getPendingAnswerHandler`` instead of ``answerHandlerRetriever``? Does that work? Ditto above & for questions.

Reply:
Yep! Changed :)